spec/rack/test/cookie_spec.rb

```diff
@@ -45,7 +70,7 @@
     it 'allow symbol access' do
       jar = Rack::Test::CookieJar.new
       jar['value'] = 'foo;abc'
-      jar[:value].should == 'foo;abc'
+      expect(jar[:value]).to eql('foo;abc')
```

Comment:
Prefer `eq` to `eql`, since that's what's being used elsewhere in the code. (_unless_ it actually causes a different result, but I don't think it should in this case)